sandbox/registration-processor-mz.properties

```diff
@@ -381,7 +381,7 @@ mosip.regproc.camel.bridge.eventbus.kafka.poll.frequency=100
 #MosipVerticleAPIManager
 mosip.regproc.camel.bridge.eventbus.kafka.group.id=camel-brdige-mz
 
-mosip.regproc.camelbridge.pause-settings=[{"matchExpression": "$.tags[?(@['AGE_GROUP']== 'CHILD' && @['ID_OBJECT-residenceStatus'] == 'nonResident')]","pauseFor": 600,"defaultResumeAction": "RESUME_PROCESSING","fromAddress": "eventbus://packet-classifier-new-bus-out","resumeRemoveTags" : ""},{"matchExpression": "$.tags[?(@['HOTLISTED'] == 'operator')]","pauseFor": 432000,"defaultResumeAction": "STOP_PROCESSING","fromAddress": ".*","resumeRemoveTags" : "HOTLISTED"}]
+mosip.regproc.camelbridge.pause-settings=[{"ruleId" :"NON_RESIDENT_CHILD_APPLICANT","matchExpression": "$.tags[?(@['AGE_GROUP']== 'CHILD' && @['ID_OBJECT-residenceStatus'] == 'nonResident')]","pauseFor": 600,"defaultResumeAction": "RESUME_PROCESSING","fromAddress": "eventbus://packet-classifier-new-bus-out","ruleDescription" : "Non resident child applicant paused"},{"ruleId" :"HOTLISTED_OPERATOR","matchExpression": "$.tags[?(@['HOTLISTED'] == 'operator')]","pauseFor": 432000,"defaultResumeAction": "STOP_PROCESSING","fromAddress": ".*","ruleDescription" : "HotListed paused"}]
```

Comment:
for first rule change to "Non resident child applicant packet"
for second rule, change to "Packet created by hotlisted operator"

Reply:
fixed